### ISSUE TITLE
[Bullet] GridMap collisions broke from 3.1.2 to 3.2

### ISSUE THREAD

**Issue by trollodel** [godotengine/godot#35713]
**Godot version:**
3.2

**OS/device including version:**
Ubuntu 18.04 based distro (KDE neon)

**Issue description:**
I have a GridMap with a MeshLibrary where KinematicBody entities walk on (using move_and_collide).
In 3.1.2 these enemies walk correctly, without colliding with the Gridmap.
But in 3.2, enemies collide with the Gridmap and can't move across the tile's CollsionShape edges.
Only after some time, the enemies can across the tile, but they will be blocked by another tile.
However, they can't ever pass across a vertex.

Here is a sample images with the collision shapes. The enemy's CollsisionShape is the cylinder. 
![bug godot](https://user-images.githubusercontent.com/33117082/73390317-20dcd780-42d6-11ea-804d-dbee65972afd.png)

Ask me for any specific question about the movement or something else.

**Comment by Chaosus** [comment]
`Godot version: 7.2`

Are you from the future?

**Comment by Calinou** [comment]
@trollodel Please upload a minimal reproduction project to make this easier to troubleshoot.

**Comment by trollodel** [comment]
Here is a slimmed down version of my project
[gridmap-example.zip](https://github.com/godotengine/godot/files/4141297/gridmap-example.zip)

There is a KinematicBody that follows the player (Use WASD to move the player ).
The project has no sprites, so you should enable collision shapes visibility.

**Comment by pouleyKetchoupp** [comment]
Can be still reproduced in 3.2.4 beta 5, but occurs only when using Bullet Physics (default settings). Switching to Godot Physics 3D fixes the movement issues (although cylinders are not supported in Godot Physics yet, so I had to switch to boxes to check).

**Comment by akien-mga** [comment]
Duplicate of #35780, fixed by #56801.
